tools/gen-wire-tests/juju.config

```diff
@@ -19,6 +19,7 @@ folders:
     - ck
   skip-lxd:
     - caasadmission
+    - constraints
```

Comment:
we do want to run this on lxd as well so remove this line and regenerate

Reply:
for now, we have constraints test only for aws (as was in old python-based).

If we wish, we need firstly add new tests in the constraints suite (now it has only "run_contstraints_aws")